godtools/App/Features/Dashboard/Domain/GetShouldShowLanguageSettingsBarButtonUseCase.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  GetShouldShowLanguageSettingsBarButtonUseCase.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 10/24/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+
+class GetShouldShowLanguageSettingsBarButtonUseCase {
```

Comment:
Hey @rachaelblue can you move this class into a folder ```GetShouldShowLanguageSettingsBarButtonUseCase```.    I'd like each UseCase to have its own directory in the event we add additional domain models with it.